src/components/UnmanagedInterfacesList.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) [2020] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import cockpit from "cockpit";
+import React, { useState, useEffect, useCallback } from 'react';
+import { Card, CardBody } from '@patternfly/react-core';
+import {
+    Table,
+    TableBody,
+    TableHeader,
+    TableVariant,
+    cellWidth,
+    truncate
+} from '@patternfly/react-table';
+import interfaceType from '../lib/model/interfaceType';
+import interfaceStatus from '../lib/model/interfaceStatus';
+
+const _ = cockpit.gettext;
+
+const UnmanagedInterfacesList = ({ interfaces = [], connections = [] }) => {
```

Comment:
Just a question, why do we need to have two quite-similar components instead of using a prop to give it the interfaces?

Reply:
I added it by now just by convenience, but I guess we could just use one and pass a managed prop. That will imply a lot of conditional about the information to show I guess

Reply:
As I said, it was just a question :) According to your answer, I'm not sure if it pay off unifying them right now. So, don't worry and forget my previous comment.

Reply:
I am not sure whether it may have lots of conditionals because the main difference is that this new list will not display interface details. But I think it makes sense to have different components too. For instance, the UnmanagedInterfacesList does not get the `connections` list.

Reply:
Yep, that is the main difference by now.